airflow/api_fastapi/views/ui/assets.py

```diff
@@ -30,7 +30,7 @@
 assets_router = AirflowRouter(tags=["Asset"])
 
 
-@assets_router.get("/next_run_datasets/{dag_id}", include_in_schema=False)
+@assets_router.get("/next_run_assets/{dag_id}", include_in_schema=False)
```

Comment:
Should we update the return data in this file too? `dataset_expression` -> `asset_expression`?

Reply:
This is also a db column in DAG model. I intend to change it in https://github.com/apache/airflow/pull/42023